wscript

```diff
@@ -80,17 +87,24 @@ def configure(conf):
   conf.check_cxx(lib = 'msgpack')
   conf.check_cxx(lib = 'jubatus_mpio')
   conf.check_cxx(lib = 'jubatus_msgpack-rpc')
-  conf.check_cxx(lib = 'dl')
+  if Options.options.freebsd:
+    conf.check_cxx(lib = 'jubatus_core')
+    conf.check_cxx(lib = 'log4cxx')
+    conf.env.LINKFLAGS += ['-ljubatus_core']
+    conf.env.LINKFLAGS += ['-llog4cxx']
+  else:
+    conf.check_cxx(lib = 'dl')
 
   # pkg-config tests
   conf.find_program('pkg-config') # make sure that pkg-config command exists
-  try:
-    conf.check_cfg(package = 'liblog4cxx', args = '--cflags --libs')
-    conf.check_cfg(package = 'jubatus_core', args = '--cflags --libs')
-  except conf.errors.ConfigurationError:
-    e = sys.exc_info()[1]
-    conf.to_log("PKG_CONFIG_PATH: " + os.environ.get('PKG_CONFIG_PATH', ''))
-    conf.fatal("Failed to find the library. Please confirm that PKG_CONFIG_PATH environment variable is correctly set.", e)
+  if not Options.options.freebsd:
```

Comment:
I think `pkg-config` works fine in FreeBSD, too.

```sh
%> PKG_CONFIG_PATH=/usr/local/libdata/pkgconfig pkg-config liblog4cxx --cflags --libs   
-I/usr/local/include -L/usr/local/lib -llog4cxx  
```

As I don't think any gcc extentions are not use, it must compile with clang too.